usage-advanced.py

```diff
@@ -1,136 +1,221 @@
                 ),
-                dcc.Slider(
-                    id="slider-intervalSecondsLoaded",
-                    min=200,
-                    max=2000,
-                    step=None,
-                    updatemode="drag",
-                    marks={i: str(i) for i in [200, 500, 750, 1000, 2000]},
-                    value=500,
-                ),
-                html.P("Update Interval for duration:", style={"margin-top": "30px"}),
-                dcc.Slider(
-                    id="slider-intervalDuration",
-                    min=200,
-                    max=2000,
-                    step=None,
-                    updatemode="drag",
-                    marks={i: str(i) for i in [200, 500, 750, 1000, 2000]},
-                    value=500,
-                ),
-                html.P(
-                    "Seek To (% of total media length):", style={"margin-top": "30px"}
-                ),
-                dcc.Slider(
-                    id="slider-seek-to",
-                    min=0,
-                    max=1,
-                    step=None,
-                    updatemode="drag",
-                    marks={i: str(i * 100) + "%" for i in [0, 0.25, 0.5, 0.75, 1]},
-                    value=0,
+                html.Div(
+                    style={"width": "45%", "padding": "10px"},
+                    children=[
+                        html.Div(
+                            [
+                                dcc.Checklist(
+                                    id="radio-bool-props",
+                                    options=[
+                                        {"label": val.capitalize(), "value": val}
+                                        for val in [
+                                            "playing",
+                                            "loop",
+                                            "controls",
+                                            "muted",
+                                        ]
+                                    ],
+                                    value=["controls"],
+                                ),
+                                html.Div(
+                                    [
+                                        dcc.Input(
+                                            id="seekto-input",
+                                            type="number",
+                                            placeholder="seekTo value",
+                                        ),
+                                        html.Button("seekTo", id="seekto-btn"),
+                                    ],
+                                ),
+                            ],
+                            style={
+                                "display": "flex",
+                                "flex-direction": "row",
+                                "justify-content": "space-between",
+                                "margin-bottom": "5px",
```

Comment:
Style keys should be camelCase, right? ie `flexDirection` instead of `flex-direction`

Reply:
it takes either but I've found some weird edge cases where only hyphenated works instead of camelCase so I stick with the former

Reply:
Really? When I've used kebab-case seems like it works but gives tons of console warnings. Do you have a record of where you've had problems with this? Anyway in https://dash.plotly.com/dash-html-components we tell people:
> Properties in the `style` dictionary are camelCased

So if that doesn't work consistently we need to know about it!!

Reply:
Right, fair point about the console warnings - I just converted everything here to camelCase. I'll definitely flag issues if I see issues with camelCase in the future.